src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,46 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.health = health;
+    this.name = name;
+
+    Animal.alive.push(this);
+  }
 }
+Animal.alive = [];
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health = 100, hidden = false) {
+    super(name, health);
+    this.name = name;
+    this.hidden = hidden;
+    this.health = health;
+
+    Animal.alive.push(this);
```

Comment:
you can write it only once in the Animal class

Reply:
Yes, I just tested something, and forgot to remove this)